platformio.ini

```diff
@@ -1351,6 +1351,26 @@ extra_scripts        = ${common.extra_scripts}
   buildroot/share/PlatformIO/scripts/stm32_bootloader.py
   buildroot/share/PlatformIO/scripts/mks_encrypt.py
 
+#
+# MKS Robin Pro V2
+#
+[env:mks_robin_pro2]
```

Comment:
Was the new board supposed to be in a different PR? It was not mentioned in the description.

Reply:
I didn't put it in a separated PR because this a never-will-be-released board used only for development purposes. I pushed it because (1) I used it for F4 and because it is currently the first start point for *any* F4 board with TFT support. (we may use it as reference for the new Anet).